src/development/tools/devtools/memory.md

```diff
@@ -12,6 +12,19 @@ the first section explains how Dart manages memory.
 If you already understand Dart’s memory management,
 you can skip to the [Memory view guide](#memory-view-guide).
 
+## When the memory screen is needed
+
+Use the memory view if any of the following conditions occur:
+
+* Your application experiences out-of-memory crashes.
+* Your application slows down.
+* Your application causes the device to slow down or become unresponsive.
+* OS kills your application because of memory over-usage.
+* Your app memory usage is larger than the acceptable limit. 
+This limit can vary depending on the type of devices your app targets.
+* You aren’t sure whether your feature is managing memory properly and 
+might be leaking.
```

Comment:
**issues:** Some concerns with this copy:

- The headline does not use syntax parallel with the other headings. It uses a dependent clause structure as opposed to those elsewhere that you short noun phrases. Consider `Reasons to use the Memory page` or something similar.
- Most, if not all, of these list items could start with "Your application...", so I would recommend changing the preceding text to something like:
  `Use the Memory page when your application experiences one of the following conditions:`
- The list lacks [parallel syntax](https://developers.google.com/style/lists#parallel-syntax) as well. Consider setting to a verb phrase that builds off of the introductory sentence. For example:
```markdown
* Crashes when it runs out of memory 
* Slows down
* Makes the device on which it's installed run slower or become unresponsive
* Shuts down because it exceeded the operating system memory usage limit
...
```

Reply:
Thanks for the suggestion.